docs/user/Alerts.md

```diff
@@ -33,6 +33,25 @@ Machine did not reach the “Running” Phase.  Running phase is when the machin
 ### Resolution
 If the machine never became a node, consult the machine troubleshooting guide.
 
+## MachineWithOldDeletionTimestamp
+Machine has been in the "Deleting" phase for a long time. Deleting phase is added to a machine when it has been marked for deletion and given a deletion timestamp in etcd.
+
+### Query
+```
+# for: 180m
+(mapi_machine_created_timestamp_seconds{phase=="Deleting"}) > 0
+```
+
+### Possible Causes
+* Invalid cloud credentials are preventing deletion.
+* A [Pod disruption budget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets) is
+  preventing Node removal.
+
+### Resolution
+Consult the `machine-controller`'s logs for root causes (see the [Troubleshooting Guide](TroubleShooting.md). In some
```

Comment:
We should work to ensure in the future that looking at the logs is not a necessity.  We should prioritize putting the information on the machine's status.